### PR TITLE
Removes alcohol overdose

### LINKED ISSUE #7115: Alcohol is extremely toxic
(WEB REPORT BY: senaat1 REMOTE: 158.69.55.2:7777)
Revision: b562e7a976a28f0bf87ae206f550ce51c9ea2c84

After drinking three sips from a fresh, untouched bottle of rum i dispensed from the ghetto bar's booze-o-mat my health quickly dropped into orange/dark orange territory and i got VERY drunk. Can we just remove alcohol doing any damage whatsoever? This is almost as bad as Leroy_Lightyear's cancer was.

### PR DESCRIPTION
**Pull request by clusterfack**
This is distinct from removing alcohol's toxicity. Alcohol poisons you in 4 stages each with varying effects of slurring speech up to damaging your liver and you with toxins. However deadrum/absinthe (which covers almost all alcohols) will begin toxing you above 30u, the reagents_overdose point, whereas normal alcohol shouldn't be damaging you until 450u.

This PR instead removes >30u toxins damage, and to compensate lowers the pass_out (normal toxins damage level) to 350u for variants of deadrum/absinthe instead while increasing the tox damage at that level.

Closes #7129
Fixes #7115

Arthur wasn't making my suggested changes quickly enough/at all

### CODE REVIEW

**Comment by DrCelt** [comment]
literally copying #7189. also this fixes #7115

**Comment by 9600bauds** [comment]
I mean I'm all for alcohol not being deadly but 450 units? Am I reading that right?
You need 450 units of alcohol IN YOUR SYSTEM AT ONCE for there to be any ill effect?

**Comment by clusterfack** [comment]
That's how it used to work for every alcohol. The normal system would not damage you or your liver properly until 450 units. All the damage you thought was normal alcohol toxins damage was in fact, reagents overdose.

**Comment by DrCelt** [comment]
450 units of alcohol is more units of alcohol in the blood than there's blood, pretty sure.

**Comment by clusterfack** [comment]
Theres 560u of blood so close but not quite

**Comment by 9600bauds** [comment]
You also process around 0.7 units of drinks per second so you would need to ingest way more than 450u just to build up that amount in your system at once.

**Comment by DrCelt** [comment]
Need a changelog, also a tag with controversial

**Comment by Intigracy** [comment]
:+1: but you should lower it to like 200 so liver damage remains something that exists.

**Comment by clusterfack** [comment]
I can lower certain variants to that if you want, let me know which of the types you consider the most absolutely lethal and I will

**Comment by Intigracy** [comment]
I think that should be the highest number honestly, with others possibly being lower.

The only way you're getting 200 units in you is if you're drinking the entire bar dry.

**Comment by clusterfack** [comment]
Well I'd prefer some variance in the final pass out level because if you have multiple alcohols they can still semi stack if they are all at the same values for each stage, which is another reason I wanted to distinguish them.

**Comment by Intigracy** [comment]
In order for highest to lowest alchohol content commonly:

Absinthe
Whiskey
Gin
Rum
Brandy
Vodka
Wine
Mead
Malt Liquor
Beer
Cider
Light Beer

Stolen from https://en.wikipedia.org/wiki/Alcohol_by_volume#Typical_levels

**Comment by clusterfack** [comment]
Alright, changed the values from 175 for absinthe, to about 250 for whiskeys/rums, to 325 for the regular deadrums

**Comment by Intigracy** [comment]
:)

**Comment by SonixApache** [comment]
:+1:

**Comment by DrCelt** [comment]
The real issue is that alcohol should be its own chemical and drinks containing it instead, this just skits around the issue.

**Comment by Arthurdentist** [comment]
You shouldn't lower the liver damage threshold at all. 

200u = 4 full glasses of some cocktail which isn't exactly hard for a bartender to do. Drinking yourself to death should take some work. You should need to be committed to the idea.

**Comment by Kurfursten** [comment]
@clusterfack I hope you remembered to change sip size. Put it around 2-4u, like bitesize.

11u is way too much

**Comment by clusterfack** [comment]
@Arthurdentist I shouldn't lower it, instead I should just remove it entirely as your PR does?

@Kurfursten If someone wants to do that or everyone here wants it sure but if I start bundling a lot of amendments into this that others don't agree with I don't think it's going to go anywhere.

**Comment by Arthurdentist** [comment]
I'd rather go back to having no liver damage trouble at all (which is what my PR does). Failing that I'd prefer it if it stayed as it currently is and the issue of rum being deadly as fuck was just resolved.

**Comment by Intigracy** [comment]
But this does address the issue of rum being deadly as fuck.

You no longer take damage from 30+ units, you only start taking damage from alchohol once you hit the thresholds in this PR. And if you're at that level of alchohol in your system, then you're downing too much too fast.

**Comment by ririchiyo** [comment]
Even when the liver begins to take damage, it needs to sustain damage up until it reaches 10 damage in order to begin to even harm you. So, at worst it would need to have 20 ticks of > liver damage value in order to cause serious harm to your body that would become deadly later on.

**Comment by Intigracy** [comment]
This pr is great.

**Comment by 9600bauds** [comment]
I haven't actually tested this out in-game, so I'll ask here:
Do you actually need 450 units of booze in your system AT ONCE for any ill effect to take place?
Even 250 units of whiskey is a goddamn achievement if I'm reading this right, consider also that you absorb 0.7 units per second and the math checks out at about 3 full bottles of whiskey in quick succession for even a tiny bit of damage to start taking place.

**Comment by clusterfack** [comment]
No because right now you take 1 toxin damage per tick over 30u

**Comment by FudgePucker** [comment]
Maybe make alcoholic drinks break down into ethonal and then have that do the damage.

**Comment by FudgePucker** [comment]
That way, antitoxin will turn into antihol to help treat people who drank two bottles of whisky.

**Comment by Kurfursten** [comment]
That seems to just dumb down the treatment of alcohol which is not actually hard at all. If they can get to medbay at all they'll be fine: ethylox instantly turns booze in your system to water and cures the effects.

**Comment by clusterfack** [comment]
How exactly does it dumb it down? Or rather, what part of the old system made it have 'more depth'?

**Comment by Kurfursten** [comment]
Antihol is the TGstation name for Ethylredox, it sounds like he wants to change the recipe for Ethyl from antitox + oxygen + carbon to antitox+ethanol which would make the treatment of alcohol poisoning much easier: a doctor doesn't need to identify the patient has an OD of alcohol, they just need to stuff "green pill in mouth for green damage on analyzer"

Maybe I just misunderstand his statement. Feel free to clarify.

**Comment by clusterfack** [comment]
Oh I thought you were addressing the overall PR, I hadn't read his post

**Comment by Intigracy** [comment]
merge when

**Comment by FudgePucker** [comment]
I am retarded kurf, I was thinking of goon chem. My bad.

**Comment by Intigracy** [comment]
REBASE IT REEEEEEEEEEEEEE

**Comment by clusterfack** [comment]
Alright
